src/ophys_etl/pipelines/segment_binarize_pipeline.py

```diff
@@ -0,0 +1,54 @@
+import argschema
+from ophys_etl.transforms.suite2p_wrapper import Suite2PWrapper
+from ophys_etl.transforms.convert_rois import BinarizerAndROICreator
+import tempfile
+import pathlib
+import json
+
+
+class SegmentBinarizeSchema(argschema.ArgSchema):
+    motion_corrected_video = argschema.fields.InputFile(
```

Comment:
I think you can use the `H5InputFile` field that Kat wrote in:

https://github.com/AllenInstitute/ophys_etl_pipelines/blob/master/src/ophys_etl/schemas/fields.py

Reply:
done.